### DISCUSSION TITLE
Does not work with Portainer, Docker desktop

### DISCUSSION THREAD

**Discussion by ghost** [nextcloud/all-in-one#932] [Questions] [answered]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are affected by the same issue.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Steps to reproduce
1. create container in portainer
2. Container stops immediately
3. Error message: Docker socket is not available. Cannot continue.

### Expected behavior 
Container should be created and running

### Actual behavior 
Container is created but stops immediately

**Comment by szaimen** [comment]
which OS and can you show what you've entered in Portainer?

**Comment by ghost** [comment]
> which OS and can you show what you've entered in Portainer?

Ubuntu server 22.04

![grafik](https://user-images.githubusercontent.com/90777102/178824249-5c25a473-8dfb-4da2-b4a0-8ad8ecb7db4e.png)

**Reply by szaimen** [reply]
Obviously you are missing the bind-mounts/volumes and please remove 3478 UPD and TCP from this container!

**Reply by ghost** [reply]
Why should these ports not be open for this container? Is this only the master container? Please, Excuse the not knowing.

**Reply by szaimen** [reply]
Because the talk container will use these.

But anyways, good that you have found a solution with Portainer Stacks and the docker-compose file!

**Reply by ghost** [reply]
Yes great thing these compose files😆. I think this is the right time to thank you for the great project. AIO makes it so much easier and in case of doubt safer to run a Nextcloud. @szaimen Thanks a lot!

**Reply by szaimen** [reply]
Thanks for the kind words! 😊

**Comment by Zoey2936** [comment, marked as answer]
I would recommend the stacks (docker-compose) function of portainer and using the content of this docker-compose file:
https://github.com/nextcloud/all-in-one/blob/main/docker-compose.yml

**Reply by szaimen** [reply]
good idea!

**Reply by ghost** [reply]
Yes, it works very well.
